go/stats/config.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/inverse-inc/packetfence/go/pfconfigdriver"
+)
+
+// readDBConfig read pfconfig database configuration
+func readDBConfig() pfconfigdriver.PfConfDatabase {
```

Comment:
Use the database package instead to avoid code duplication

Reply:
Replicated from: https://github.com/inverse-inc/packetfence/blob/devel/go/dhcp/config.go#L57

I'm not sure what you mean. Do we want a separate struct from https://github.com/inverse-inc/packetfence/blob/devel/go/pfconfigdriver/structs.go#L110 within the stats package?

Reply:
Here is the proper way to connect to the DB, the code in dhcp/ was missed from my refactor.

```
import "github.com/inverse-inc/packetfence/go/database"
ctx := context.Background() // Or your current context
pfconfigdriver.PfconfigPool.AddStruct(ctx, &pfconfigdriver.Config.PfConf.Database)
database := database.ConnectFromConfig(pfconfigdriver.Config.PfConf.Database)
```

That way we don't duplicate calls to pfconfig and have a ton of readDBConfig methods everywhere, and it allows to configure the connection options in a single place